felayout_t3kit/dev/styles/main/contentElements/heroImage.less

```diff
@@ -9,6 +9,8 @@
     background-repeat: no-repeat;
     width: 100%;
     position: relative;
+    opacity: 0;
```

Comment:
I totally don't understand why you hide Hero Image. Looks like a new potential bug?